src/control/cmd/daos_server/storage.go

```diff
@@ -47,20 +50,24 @@ func (es *errs) add(err error) error {
 	return nil
 }
 
+func iommuDetected() bool {
+	// Simple test for now -- if the path exists and contains
+	// DMAR entries, we assume that's good enough.
+	dmars, err := ioutil.ReadDir(iommuPath)
+	if err != nil {
+		return false
+	}
+
+	return len(dmars) > 0
+}
```

Comment:
Out of curiosity, given that this code imports the server package anyhow, why not export the helper from that package?

Reply:
I wanted advice on where to move it to and @kjacque has provided that, moving it to lib/lhardware

Reply:
done